docs/.vuepress/config.js

```diff
@@ -119,7 +119,11 @@ module.exports = {
           {
             text: "C++ SDK",
             link: "https://docs.cronos.org/cronos-play/getting-started_cpp",           
-          }
+          },
+          {
+            text: "CDC Pay Integration",
+            link: "https://docs.cronos.org/cronos-play/cdc-pay-integration",           
+          },
```

Comment:
I think we can first merge to master, but just for future reference, it's preferred to make a PR against gitbook branch, and update the `SUMMARY.md` file instead, as otherwise we'd have to do double work